login-utils/lslogins.c

```diff
@@ -1494,6 +1558,7 @@ int main(int argc, char *argv[])
 		OPT_WTMP = CHAR_MAX + 1,
 		OPT_BTMP,
 		OPT_LASTLOG,
+		OPT_LASTLOG2,
```

Comment:
Maye also add `#ifdef HAVE_LASTLOG2` if all related to the command line option is also #ifdef-ed ;-)